src/main/kotlin/ui/strings/Strings.kt

```diff
@@ -12,7 +12,7 @@ enum class Strings(
     val ja: String,
     val zhCN: String,
     val ru: String = en,
-    val fr: String = en
+    val fr: String
```

Comment:
Please keep it so it allows fallback to English text before localized texts are available.
Otherwise the build cannot pass.